documents/knowledge/Elements.txt

```diff
@@ -2118,7 +2118,7 @@ k□ (Directions)
     num a, num b: roots(a, b) / x^2 + ax + b = 0
     num a, str b: evaluate single variable expression b with x=a
     str a, num b: evaluate single variable expression a with x=b
-    str a, str b: solve equation a < b for x
+    str a, str b: solve equations a and b simultaneously
```

Comment:
The "simultaneously" here sounds a teeny bit odd. Maybe clarify that they're a system of equations or something like that?

Reply:
Where I'm from, they're called simultaneous equations, so that's why I used that term,